src/components/Tile.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Tile({image, imageAlt, title, paragraphs }) {
+                if (image) {
+                   return (<section>
+                   <img src={image} alt={imageAlt}/>
+                   </section>)
+                } else {return (<section>
+                    <h2>{title}</h2>
```

Comment:
hier is zoals je inmiddels weet een veel simpelere oplossing voor met {children}